sdk/translation/Azure.AI.Translation.Document/tests/ContainerDocumentTranslationClientTests.cs

```diff
@@ -47,5 +48,24 @@ public async Task StartSynchronousDocumentTranslation()
             Assert.IsNotNull(responseString);
             Assert.AreNotEqual(requestString, responseString);
         }
+
+        //It is for serialization sanity test, not for functional test
+        [Test]
+        public void DocumentTranslationSerializationTest()
+        {
+            string filePath = Path.Combine("TestData", "test-input-2.txt");
+            using Stream fileStream = File.OpenRead(filePath);
+            var sourceDocument = new MultipartFormFileData(Path.GetFileName(filePath), fileStream, "text/html");
+
+            filePath = Path.Combine("TestData", "test-glossary-2.csv");
+            using Stream glossaryStream = File.OpenRead(filePath);
+            var sourceGlossaries = new List<MultipartFormFileData>()
+            {
+                new(Path.GetFileName(filePath), glossaryStream, "text/csv")
+            };
+            DocumentTranslateContent content = new DocumentTranslateContent(sourceDocument, sourceGlossaries);
+            BinaryData result = ModelReaderWriter.Write<DocumentTranslateContent>(content);
+            Assert.IsNotNull(result);
```

Comment:
so actually this `ModelReaderWriter.Write<T>` method never returns a null - could we check what is inside the content?

I kind of do not like that we have this test. In my original comment, I am suggesting that we just have a local test verifying everything is expected and we are good to go.